### PR TITLE
[issue 194] Closing issues or merge requests does not create an event post with a link back to the project #194

### LINKED ISSUE #194: Closing issues or merge requests does not create an event post with a link back to the project
When closing an issue or merge request, when the event is delivered rather then a link to the the project, the post will include text `[dhaussermann/dkh-project3]`  

RepoLink

This issue is not new. I have just noticed that these events seem to differ from several other events where a proper link is included. 

These posts should match the new issue and new pull requests event (shown above) with a proper link.

### PR DESCRIPTION
**Pull request by sibasankarnayak**
added proper link on notification on PR (close , open, reopen) and issue (close, repoen)

ticket here

Fixes #194

### CODE REVIEW

File: server/webhook/issue.go [hunk 1]
```diff
@@ -74,9 +74,9 @@ func (w *webhook) handleChannelIssue(event *gitlab.IssueEvent) ([]*HandleWebhook
 	case actionOpen:
 		message = fmt.Sprintf("#### %s\n##### [%s#%v](%s)\n###### new issue by [%s](%s) on [%s](%s)\n\n%s", issue.Title, repo.PathWithNamespace, issue.IID, issue.URL, senderGitlabUsername, w.gitlabRetreiver.GetUserURL(senderGitlabUsername), issue.CreatedAt, issue.URL, issue.Description)
 	case actionClose:
-		message = fmt.Sprintf("[%s] Issue [%s](%s) closed by [%s](%s)", repo.PathWithNamespace, issue.Title, issue.URL, senderGitlabUsername, w.gitlabRetreiver.GetUserURL(senderGitlabUsername))
+		message = fmt.Sprintf("[%s](%s) Issue [%s](%s) closed by [%s](%s)", repo.PathWithNamespace, repo.GitHTTPURL, issue.Title, issue.URL, senderGitlabUsername, w.gitlabRetreiver.GetUserURL(senderGitlabUsername))
```
Comment (mickmister) [inline server/webhook/issue.go:77]: `GitHTTPURL` seems like the wrong one to use here. Let's use `WebURL` instead

**Review by mickmister — CHANGES_REQUESTED** [review]

**Review by mickmister — APPROVED** [review]

**Review by hanzei — APPROVED** [review]

**Comment by hanzei** [comment]
@DHaussermann Gentle reminder to review the PR

**Review by DHaussermann — APPROVED** [review]
Tested and passed
For both issues and PRs - the project is now a link.
Ensured links are functional

LGTM!

**Pull request merged**